SwanContents/swancontents/swanclassic/handlers/notebookviewer.py

```diff
@@ -0,0 +1,79 @@
+from tornado import web
+from jupyter_server.base.handlers import JupyterHandler, FilesRedirectHandler
+from jupyter_server.extension.handler import (
+    ExtensionHandlerMixin,
+    ExtensionHandlerJinjaMixin
+)
+from jupyter_server.base.handlers import path_regex
+from nbconvert import HTMLExporter
+
+has_voila = False
+try:
+    import voila
+    has_voila = True
+except:
+    pass
+
+
+class NotebookViewerHandler(ExtensionHandlerJinjaMixin, ExtensionHandlerMixin, JupyterHandler):
+    """
+        Jupyter server extension to provide a view-only mode to open notebooks.
+        When users receive a shared project, now they can open it and use this extension
+        to load the notebooks in view-mode (without the editing options that would allow
+        the them to edit the document, even without permissions to save the changes).
```

Comment:
`the them` -> `them`

Reply:
This commit was just a move of the file xD the text was incorrect in the code already. Should I create a different commit or combine in this?

Reply:
Just combining is fine.